src/seedu/addressbook/commands/Command.java

```diff
@@ -26,6 +26,14 @@ public Command(int targetIndex) {
     protected Command() {
     }
 
+    /**
+     *
+     * @return true if the command mutate data
+     */
+    public boolean isMutating() {
+        return true;
```

Comment:
We shouldn't assign a default behavior to the parent class even though its generally true/false
E.g. what if we assign `canFly()` to be true by default and we forgot to override it for some animals such as penguin?
result -> suddenly penguin can fly